forch/faucet_state_collector.py

```diff
@@ -363,6 +364,11 @@ def _update_dataplane_detail(self, dplane_state):
             state = State.broken
             detail.append("broken links: " + str(broken_links))
 
+        broken_vlans = self._get_broken_vlans(dplane_state)
+        if broken_vlans:
+            state = State.broken
+            detail.append('broken vlans: ' + str(broken_vlans))
```

Comment:
is this tested anywhere in the integration tests?  Seems like we should find a (simple) way to do this for just one part.  E.g., just set the threshold really low for one bit and make sure the system reports it as an error.... ?  Seems like we should be able to work it into the existing tests so as not to increase overall duration or test length. -- I was thinking somewhere in testing/test_stack

Reply:
It is tested in test_access. What it does is generating udp packets at 2pkt/s. The threshold is set to 1 pkt/s. But this test seems not working. I am looking into it.

Reply:
I think the test is working now. Can we leave the rate test in test_acces, or should we move it to test_stack?